scripts/setup_juno_optee.sh

```diff
@@ -0,0 +1,2485 @@
+#!/bin/bash
+################################################################################
+# EDIT so these match your credentials                                         #
+################################################################################
+DEV_PATH=$PWD/devel/juno_optee
+
+# You only need to set these variables if you have access to the OPTEE_TEST
+# (requires a Linaro account and access to the git called optee_test.git)
+# If, in addition to the base OPTEE_TEST, you have access to the GlobalPlatform
+# "TEE Initial Configuration" test suite, you may add the tests by extracting
+# the test package in the current directory and run:
+# $ export CFG_GP_PACKAGE_PATH=<path to the test suite directory>
+# $ export CFG_GP_TESTSUITE_ENABLE=y
+# $ ./setup_juno_optee.sh
+LINARO_USERNAME=cedric.chaumont # Should _NOT_ contain @linaro.org.
```

Comment:
- line to be commented with #
- cedric.chaumont to be replaced by firstname.lastname